src/pocket/PocketALBApplication.ts

```diff
@@ -685,8 +680,141 @@ export class PocketALBApplication extends Resource {
-      dashboardName: `${prefix}-ALBDashboard`,
+      dashboardName: `${this.config.prefix}-ALBDashboard`,
       dashboardBody: JSON.stringify(dashboardJSON),
     });
   }
+
+  private createCloudwatchAlarms(): void {
+    const defaultAlarms: CloudwatchMetricAlarmConfig[] = [
+      {
+        alarmName: `${this.config.prefix}-Alarm-HTTP5xxErrorRate`,
+        metricQuery: [
+          {
+            id: 'requests',
+            metric: [
+              {
+                metricName: 'RequestCount',
+                namespace: 'AWS/ApplicationELB',
+                period: 60,
+                stat: 'Sum',
+                unit: 'Count',
+                dimensions: { LoadBalancer: this.alb.alb.arnSuffix },
+              },
+            ],
+          },
+          {
+            id: 'errors',
+            metric: [
+              {
+                metricName: 'HTTPCode_ELB_5XX_Count',
+                namespace: 'AWS/ApplicationELB',
+                period: 60,
+                stat: 'Sum',
+                unit: 'Count',
+                dimensions: { LoadBalancer: this.alb.alb.arnSuffix },
+              },
+            ],
+          },
+          {
+            id: 'expression',
+            expression: 'errors/requests*100',
+            label: 'HTTP 5xx Error Rate',
+            returnData: true,
+          },
+        ],
+        comparisonOperator: 'GreaterThanOrEqualToThreshold',
+        evaluationPeriods: 5,
+        threshold: this.config.alarms?.http5xxError?.threshold ?? 5,
+        insufficientDataActions: [],
+        alarmActions: this.config.alarms?.http5xxError?.actions ?? [],
+        okActions: this.config.alarms?.http5xxError?.actions ?? [],
+        tags: this.config.tags,
+      },
+      {
+        alarmName: `${this.config.prefix}-Alarm-HTTPResponseTime`,
+        namespace: 'AWS/ApplicationELB',
+        metricName: 'TargetResponseTime',
+        dimensions: { LoadBalancer: this.alb.alb.arnSuffix },
+        period: 300,
+        evaluationPeriods: 1,
+        statistic: 'Average',
+        comparisonOperator: 'GreaterThanThreshold',
+        threshold: this.config.alarms?.httpLatency?.threshold ?? 300,
+        alarmDescription: 'Detecting slower-than-average HTTP response times',
+        insufficientDataActions: [],
+        alarmActions: this.config.alarms?.httpLatency?.actions ?? [],
+        okActions: this.config.alarms?.httpLatency?.actions ?? [],
+        tags: this.config.tags,
+      },
+      {
+        alarmName: `${this.config.prefix}-Alarm-HTTPRequestCount`,
+        namespace: 'AWS/ApplicationELB',
+        metricName: 'RequestCount',
+        dimensions: { LoadBalancer: this.alb.alb.arnSuffix },
+        period: 300,
+        evaluationPeriods: 1,
+        statistic: 'Sum',
+        comparisonOperator: 'GreaterThanThreshold',
+        threshold: this.config.alarms?.httpRequestCount?.threshold ?? 500,
+        alarmDescription: 'Detecting higher than normal HTTP request count',
```

Comment:
Same comment here: I'm assuming the threshold is not the normal, but the maximum threshold allowed.